src/Common/MemoryTracker.h

```diff
@@ -258,6 +258,10 @@ class MemoryTracker
     void logPeakMemoryUsage();
 
     void debugLogBigAllocationWithoutCheck(Int64 size [[maybe_unused]]);
+    Stopwatch stopwatch;
+    size_t previous_value = 0;
```

Comment:
Having these two members looks excessive.

Reply:
These two variables are used in the function from the method 
```
void MemoryTracker::updateMemoryCredits()
{
    size_t current_value = amount;
    if (current_value > previous_value && current_value - previous_value > threshold)
    {
        size_t delta = (current_value - previous_value) * stopwatch.elapsedMicroseconds();
        ProfileEvents::increment(ProfileEvents::MemoryCredits, delta);
        previous_value = current_value;
        stopwatch.restart();
    }
}
```
in MemoryTrakcer.cpp, do u have a better that these two variables should be placed to?